spec.html

```diff
@@ -6769,9 +6769,12 @@ <h1>GetSuperBase ( )</h1>
             1. Let _envRec_ be the function Environment Record for which the method was invoked.
             1. Let _home_ be _envRec_.[[HomeObject]].
             1. If _home_ has the value *undefined*, return *undefined*.
-            1. Assert: Type(_home_) is Object.
-            1. Return ? _home_.[[GetPrototypeOf]]().
+            1. [id="step-getsuperbase-assert"] Assert: _home_ is an ordinary object.
+            1. Return ! _home_.[[GetPrototypeOf]]().
           </emu-alg>
+          <emu-note>
+            <p>The assert in step <emu-xref href="#step-getsuperbase-assert"></emu-xref> is due to no observable facility to redefine the [[HomeObject]] internal slot from ECMAScript code, rather than an intentional design decision.</p>
```

Comment:
Wordsmithing:

```suggestion
            <p>The assert in step <emu-xref href="#step-getsuperbase-assert"></emu-xref> is because there is no facility to redefine the [[HomeObject]] internal slot from ECMAScript code, rather than an intentional design decision.</p>
```

But I disagree with @ljharb; I do not think we should have this note.